---
 config.mk | 21 +++++++++++----------
 1 file changed, 11 insertions(+), 10 deletions(-)

diff --git a/config.mk b/config.mk
index 718deae..0d7181d 100644
--- a/config.mk
+++ b/config.mk
@@ -1,20 +1,21 @@
 pwd    = ${PWD}
-SHELL  = /bin/sh
-AR     = ar
+SHELL  ?= /bin/sh
+AR     ?= ar
 ARFLAGS= -cr
-CC     = gcc
-CXX    = gfortran
-GFF    = gfortran
-CXX    = gfortran
+CC     ?= gcc
+CXX    ?= gfortran
+GFF    ?= gfortran
+CXX    ?= gfortran

 PROGRAM=djangoh
 PATH=${LHAPDF}/bin:/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:./
 
 LHAPDF_LIB=$(LHAPDF)/lib
-LIBS=-L$(LHAPDF_LIB) -lLHAPDF 	
+LIBS=-L$(LHAPDF_LIB) -lLHAPDF -Wl,-rpath,"$(PREFIX)/lib" -Wl,-rpath-link,"$(PREFIX)/lib"
 
-CFFLAGS = -O -Wall -pedantic -fno-automatic  -fcheck=all -g -fbacktrace -ffpe-trap=invalid,zero
-FCFLAGS = -O -Wall -pedantic -fno-automatic  -fcheck=all -g -fbacktrace -ffpe-trap=invalid,zero
-FGFLAGS = -O -Wall -pedantic -fno-automatic  -fcheck=all -g -fbacktrace -ffpe-trap=invalid,zero
+CFFLAGS = -O -m64 -g -fno-automatic -std=legacy
+FCFLAGS = -O -m64 -g -fno-automatic -std=legacy
+FGFLAGS = -O -m64 -g -fno-automatic -std=legacy
 
 INCDIR = -I$(LHAPDF)/include 
-- 
2.34.1

