From 44769f5cad37db3cd3760a26cda08b370eda257e Mon Sep 17 00:00:00 2001
From: Marco Meyer-Conde <marco@tcu.ac.jp>
Date: Fri, 9 May 2025 17:16:50 +0900
Subject: [PATCH] lhapdf5

---
 config.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.mk b/config.mk
index dae9b62..83150a7 100644
--- a/config.mk
+++ b/config.mk
@@ -11,7 +11,7 @@ PROGRAM=djangoh
 PATH=${LHAPDF}/bin:/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:./
 
 LHAPDF_LIB=$(LHAPDF)/lib
-LIBS=-L$(LHAPDF_LIB) -lLHAPDF 	
+LIBS=-L$(LHAPDF_LIB) -lLHAPDF5
 
 CFFLAGS = -O -m64 -g -fno-automatic -std=legacy
 FCFLAGS = -O -m64 -g -fno-automatic -std=legacy
-- 
2.47.0

